dsl_parser/framework/parser.py

```diff
@@ -439,6 +439,22 @@ def _calculate_element_graph(self):
                         self.element_graph.add_edge(element, dep)
                     continue
 
+                if predicates == [sibling_predicate]:
+                    # These appear to be generally in order, which means
```

Comment:
`These`? predicates? elements?
and what being in order (what order, anyway?) has to do with anything?

Reply:
Oh right, that comment was from a previous shot at the logic, I'll update.

Reply:
Comments updated.